src/detections/rule/matchers.rs

```diff
@@ -274,7 +276,15 @@ impl LeafMatcher for DefaultMatcher {
         if !err_msges.is_empty() {
             return Err(err_msges);
         }
-        if self.pipes.len() >= 2 {
+        let n = self.pipes.len();
+        if n == 0 {
+            self.match_str_len = match select_value.as_str() {
+                None => None, // strに変換できない場合は、文字列長マッチによる比較はしない
+                Some(s) if s.contains('*') | s.contains('?') => None, //ワイルドカードを含む場合は、文字列長マッチによる比較はしない
+                Some(s) => Some(s.len()),
+            };
+        }
+        if n >= 2 {
```

Comment:
You can pass the extra condition check when n=0 by using "else if".

``` suggestion
    else if n >= 2 {
```

Reply:
Thanks for suggesting! I fixed it !